plans/confluence.tf

```diff
@@ -0,0 +1,85 @@
+data "azurerm_client_config" "current" {}
+
+# This variable is only use for resetting the mysql database password
```

Comment:
I think this should be places in `vars.tf` to maintain consistency with other variable definitions